### DISCUSSION TITLE
对于评论已删除的通知，知乎原版可以显示对方回复的是我哪条回答。希望本APP也可以显示这种信息。

### DISCUSSION THREAD

**Discussion by Coder-Saru** [zhihulite/Hydrogen#152] [General]
![IMG_20241229_162019.jpg](https://github.com/user-attachments/assets/7cd96b6e-e541-485d-9976-ee8d3302fd7e)

**Comment by huajiqaq** [comment]
这里调用的是网页 暂时没有计划转为软件渲染该页面
